app/cmd/rpc/rpc_test.go

```diff
@@ -396,12 +396,14 @@ func TestRPC_Relay(t *testing.T) {
 		panic(err)
 	}
 	aat.ApplicationSignature = hex.EncodeToString(sig)
+	payload := pocketTypes.Payload{
+		Data: expectedRequest,
+	}
 	// setup relay
 	relay := pocketTypes.Relay{
-		Payload: pocketTypes.Payload{
-			Data: expectedRequest,
-		},
-		Proof: pocketTypes.Proof{
+		Payload: payload,
+		Meta:    pocketTypes.RelayMeta{BlockHeight: 5}, // todo race condition here
```

Comment:
Can you explain the race condition here?

Reply:
Block height is now needed to send a relay, and since block and creation time are unpredictable, it makes this test have a race -> `unsynced relay`